.gitignore

```diff
@@ -6,8 +6,7 @@ node_modules
 codeception.yml
 tests/*.suite.yml
 phpunit.xml
-.env.local
-.env.testing
+.env.testing.local
```

Comment:
Wondering if it makes more sense to add .env.* instead of .env.testing.local
That would cover .env.local, .env.testing, ,env.testing.local, etc…

Reply:
@lucatume ?

Reply:
Maybe this?

```suggestion
.env.*.local
```